### ISSUE TITLE
Cart accessibility w/r/t keyboard focus

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pulfalight#917]
Some text from John Jameson, the Digital Accessibility Developer at OIT:

> For Finding Aids — I can demonstrate this for you, but if you want to try yourself — when adding something to the cart, a modal dialog overlays the screen, but keyboard focus is not transferred into the dialog. Keyboard users are kinda stuck at that point; they can’t close the dialog or find it. Can you incorporate some JS as in this example accessible modal, that transfers focus into the modal when it opens?

He references https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html as an example of an accessible modal.

**Comment by sdellis** [comment]
@tpendragon This should have been fixed with PR #618 back in January. I will reach out to John to see if he can demo this or provide a GIF.

**Comment by sdellis** [comment]
Response from John Jameson:

> Testing again – focus was transferred in, but I couldn’t see my cursor on some of the elements (Firefox needs a strong :focus outline because its default is useless). I also couldn’t focus on the (x) button. I toyed with the markup inside in and changing its internal <div> to a <span> fixed that problem; I don’t know if that means there is invalid markup or if changing the element broke a JS script that was blocking focus.
> 
> Fixing the focus issue so the close button can close would help. It might also be good to map the Escape key to close the modal.
>
